onboarding-checklist.md

```diff
@@ -0,0 +1,140 @@
+# Onboarding Checklist #
+
+## Instructions ##
+
+When someone new joins the team:
+
+1. A team member adds the new team member to the `cisagov` organization in
+GitHub.
+1. Create a new issue called "Onboard _Name_".
+1. View the raw source of this file.
+1. Copy the text after the line below to the issue.
+1. Remove any block of tasks that doesn't apply to the new team member's role
+1. Submit the issue.
+1. Assign the new person and their onboarding buddy to the issue.
+1. Move the issue into _In Progress_.
+
+---
+
+## Directions ##
+
+**Onboardee and buddy:** Try to go through your checklists in order.
+
+**Buddy:** If you can’t complete any of the items on your checklist
+personally, you are responsible for ensuring that someone with the correct
+access completes that item.
+
+## Onboardee checklist ##
+
+### Getting to know VM Fusion ###
+
+- [ ] Take notes on anything that is confusing or frustrating during your
+onboarding process.
+  - Please share this information with your buddy and/or the team so we can
+  make things better.
+  - If you notice a problem (especially with things like documentation), you
+  are more than welcome to fix it!
+  - You can also file issues and pull requests on the [template onboarding checklist]
```

Comment:
This is a complete sentence, which could end in a period.  This would make this bullet agree with the others in this block.

Reply:
Fixed in a848650149665216a068543dd1b50962d7ec2737

Reply:
You added the period, but removed `checklist]`, thus breaking the link.

Reply:
Huh, turns out it helps if you save your final changes before committing. 🙄

Reply:
8ad9ac2272f97c8062f2e23c0c6b43ddc21104de